### PR TITLE
Added HTML and Markdown output support

### PR DESCRIPTION
**Pull request by SakiiR**

### CODE REVIEW

**Comment by SakiiR** [comment]
Ready to merge if you may

**Comment by joohoi** [comment]
While I'm reviewing this, could you add a changelog entry?

**Comment by SakiiR** [comment]
Of course !

File: README.md [hunk 1]
```diff
@@ -189,6 +194,8 @@ The only dependency of ffuf is Go 1.11. No dependencies outside of Go standard l
   - Changed
     - New CLI flag: -i, dummy flag that does nothing. for compatibility with copy as curl.
     - New CLI flag: -b/--cookie, cookie data for compatibility with copy as curl.
+    - New Output format are available: HTML and Markdown table.
+    - New CLI flag: -l, shows target location of redirect responses
```
Comment (joohoi) [inline README.md]: Nice catch, I didn't realize this was missing.

File: README.md [hunk 2]
```diff
@@ -15,11 +15,16 @@ Heavily inspired by the great projects [gobuster](https://github.com/OJ/gobuster
...
- - Allows fuzzing of HTTP header values, HTTP method, POST data, and different parts of URL, including GET parameter names and values
- - Silent mode (`-s`) for clean output that's easy to use in pipes to other processes.
- - Modularized architecture that allows integration with existing toolchains with reasonable effort
- - Easy-to-add filters and matchers (they are interoperable)
+- Fast!
+- Allows fuzzing of HTTP header values, POST data, and different parts of URL, including GET parameter names and values
+- Silent mode (`-s`) for clean output that's easy to use in pipes to other processes.
+- Modularized architecture that allows integration with existing toolchains with reasonable effort
+- # Easy-to-add filters and matchers (they are interoperable)
+- Fast!
```
Comment (joohoi) [inline README.md]: There seems to be a small copy-paste mistake here apparently, some of the entries are repeated.

Comment (SakiiR) [reply README.md]: Fixed :)

**Pull request merged**